main.py

```diff
@@ -0,0 +1,39 @@
+name = input("What your name?")
+if name == "Lebron" or "Lebron James":
+    print("OH MY GOD MY GLORIOUS KING I LOVE YOU!")
+else:
+    print("Oh, ok loser.")
+
+age = int(input("How old are you?"))
+if age > 16:
+    print("Damn, old ahh jit. What was Rosa Parks really like?")
+if age < 16:
+    print("Damn, bro young ahh. You missed everything ngl. Fetus lookin ahh")
+if age == 16:
+    print("Real")
+
+food = input("What's your favorite food?")
+if food != "Chicken and Rice":
+    print("Sounds like someone needs to cut.")
+else:
+    print("Locked in.")
+
+color = input("What's your favorite color?")
+if color == "Blue" or "Red" or "Green":
```

Comment:
So, what's happening here is you are doing three boolean checks: If color is "Blue" or if "Blue" or "Green" (which are both just strings, so evaluate to true), so your other color choices aren't getting picked up. Here's how we'd want to do a multi-value check like this:

```suggestion
if color == "Blue" or color == "Red" or color == "Green":
```